### DISCUSSION TITLE
Override date_created on create?

### DISCUSSION THREAD

**Discussion by daltonsutton** [directus/directus#4339] [Archive]
Is there a way to override the date_created field when using the API? I've set my date_created field to be populated by a certain date in the past, but it keeps populating with the current date when creating a new entry. Using PostgreSQL. Is there something I'm doing wrong? Thanks in advance for any help!

**Comment by benhaynes** [comment]
If permissions allow it, I think it makes sense to allow updating these fields manually in the UPDATE request. Do you agree @rijkvanzanten ?

**Reply by rijkvanzanten** [reply]
It already does 👍🏻 

I believe what we're talking about here is that it will override a user-submitted date on CREATE because it's set to "save current date"

**Reply by daltonsutton** [reply]
I had forgotten about "save current date" on create. I just set that to "do nothing" and tried it, and it has worked while using create. Using the API to import old posts and will switch it back afterwards. Thanks for the feedback!
